getting-started-java.md

```diff
@@ -199,12 +197,11 @@ In some cases bitcoinj can generate a large number of events very fast, this is
 ## Receiving money
 
 {% highlight java %}
-kit.wallet().addEventListener(new AbstractWalletEventListener() {
+kit.wallet().addCoinsReceivedEventListener(new WalletCoinsReceivedEventListener() {
     @Override
     public void onCoinsReceived(Wallet w, Transaction tx, Coin prevBalance, Coin newBalance) {
         // Runs in the dedicated "user thread".
-        //
-        // The transaction "tx" can either be pending, or included into a block (we didn't see the broadcast).
```

Comment:
Is there a reason for removing this comment?

Reply:
You are right. I removed the comment accidentally.